src/functionalTest/resources/scenarios/RIA-2999-ftpa-appellant-allowed-after-respondent-dismissed.json

```diff
@@ -0,0 +1,64 @@
+{
+  "description": "RIA-2999 Appellant FTPA Application is allowed  after the respondent application already dismissed",
+  "request": {
+    "uri": "/asylum/ccdAboutToSubmit",
+    "credentials": "Judge",
+    "input": {
+      "id": 1007,
+      "eventId": "residentJudgeFtpaDecision",
+      "state": "ftpaSubmitted",
+      "caseData": {
+        "template": "minimal-appeal-submitted.json",
+        "replacements": {
+          "ftpaAppellantDecisionOutcomeType": "allowed",
+          "ftpaRespondentDecisionOutcomeType": "dismissed",
+          "ftpaApplicantType": "appellant",
+          "ftpaApplicantType": "respondent",
+          "ariaListingReference": "987654321",
+          "notificationsSent": [
+            {
+              "id": "1007_FTPA_APPLICATION_DECISION_HOME_OFFICE_RESPONDENT",
+              "value": "34e146c9-6050-4894-a64e-acd461a5f934"
+            }
+          ]
+        }
+      }
+    }
+  },
+  "expectation": {
+    "status": 200,
+    "errors": [],
+    "caseData": {
+      "template": "minimal-appeal-submitted.json",
+      "replacements": {
+        "notificationsSent": []
+      }
+    },
+    "notifications": [
+      {
+        "reference": "1007_FTPA_APPLICATION_DECISION_HOME_OFFICE_APPELLANT",
+        "recipient": "{$respondentEmailAddresses.respondentEvidenceDirection}",
+        "subject": "Immigration and Asylum appeal: application to appeal to Upper Tribunal",
+        "body": [
+          "PA/12345/2019",
+          "A1234567",
+          "987654321",
+          "Talha Awan"
+        ]
```

Comment:
The following needs to be included in the functional tests (and injected into the GovNotify templates) to follow the same template standard. This was introduced in RIA-2936 (currently in master):

```
"{$iaExUiFrontendUrl}",
"{$customerServices.telephoneNumber}",
"{$customerServices.emailAddress}"
```
The linkToOnlineService also needs to be a dynamic placeholder in the GovNotify templates now (i.e. not hard-coded).

Reply:
Fixed